pyaedt/hfss.py

```diff
@@ -3022,15 +3022,25 @@ def create_impedance_between_objects(
     def create_boundary(
         self, boundary_type=BoundaryType.PerfectE, sheet_name=None, boundary_name="", is_infinite_gnd=False
     ):
-        """Create a boundary given specific inputs.
+        """Assign a boundary condition to a sheet or surface. This method is generally
+           used by other methods in the ``Hfss`` class such as :meth:``Hfss.assign_febi``
+           or :meth:``Hfss.assign_radiation_boundary_to_faces``.
 
         Parameters
         ----------
-        boundary_type : str, optional
-            Boundary type object. Options are ``"Perfect E"``, ``"Perfect H"``, ``"Aperture"``, and
-            ``"Radiation"``. The default is ``PerfectE``.
+        boundary_type : int, optional
+            Specifies the type of boundary condition to be assigned to a sheet or surface.
+            Options are properties of :class:``Hfss.BoundaryType``. For example,
+                - ``Hfss.BoundaryType.PerfectE``
+                - ``Hfss.BoundaryType.PerfectH``
+                - ``Hfss.BoundaryType.Radiation``
+                - ``Hfss.BoundaryType.Impedance``
+                - ``Hfss.BoundaryType.LumpedRLC``
+                - ``Hfss.BoundaryType.FEBI``
+            The default is ``Hfss.BoundaryType.PerfectE``.
```

Comment:
```suggestion
            Type of boundary condition to assign to a sheet or surface. The
            default is ``Hfss.BoundaryType.PerfectE``. Options are the properties of the
            :class:``Hfss.BoundaryType`` class. For example:
            
                - ``Hfss.BoundaryType.PerfectE``
                - ``Hfss.BoundaryType.PerfectH``
                - ``Hfss.BoundaryType.Radiation``
                - ``Hfss.BoundaryType.Impedance``
                - ``Hfss.BoundaryType.LumpedRLC``
                - ``Hfss.BoundaryType.FEBI``

```
Brief description should be noun-like. A blank line is needed before and after a bulleted list to display properly in the doc.